app/main/views/root.py

```diff
@@ -17,3 +26,57 @@ def mci_survey():
 @main_blueprint.route('/cover-page', methods=['GET'])
 def cover_page():
     return render_template('cover-page.html')
+
+
+@main_blueprint.route('/questionnaire/<questionnaire_id>', methods=['GET', 'POST'])
+def questionnaire(questionnaire_id):
+
+    # load the schema
+    json_schema = load_schema(questionnaire_id)
+    parser = SchemaParserFactory.create_parser(json_schema)
+    schema = parser.parse()
+
+    # load the response store
+    response_store = ResponseStoreFactory.create_response_store()
+
+    # load the validation store
+    validation_store = ValidationStoreFactory.create_validation_store()
+
+    # Create the validator
+    class Validator(object):
```

Comment:
Does this class need to inside the function? Can we move it outside or separate validator module?